<launch>

  <node name="ee_position_ctrl" pkg="deformation_control" type="ee_position_ctrl" output="screen">
    <remap from="left_joint_states" to="kuka_left/joint_states_msr"/>
    <remap from="right_joint_states" to="kuka_right/joint_states_msr"/>
  </node>

</launch>
